src/main/resources/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder/tasks-top-extra.jelly

```diff
@@ -30,10 +30,10 @@ THE SOFTWARE.
     <j:if test="${computation != null}">
         <j:set var="title" value="${computation.displayName}"/>
     </j:if>
-    <l:task icon="images/24x24/notepad.png" href="${url}/computation/" title="${title}">
+    <l:task icon="images/24x24/notepad.png" href="${computation.getUrl()}" title="${title}">
```

Comment:
`computation` can not be `null`. cf. https://github.com/jenkinsci/cloudbees-folder-plugin/pull/58#discussion_r60394609